include/cupla/types.hpp

```diff
@@ -63,8 +63,7 @@ inline namespace CUPLA_ACCELERATOR_NAMESPACE
     defined(ALPAKA_ACC_CPU_B_SEQ_T_THREADS_ENABLED) ||                         \
     defined(ALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLED) ||                            \
     defined(ALPAKA_ACC_CPU_B_SEQ_T_SEQ_ENABLED) ||                             \
-    defined(ALPAKA_ACC_CPU_B_TBB_T_SEQ_ENABLED) ||                             \
-    defined(ALPAKA_ACC_CPU_BT_OMP4_ENABLED)
+    defined(ALPAKA_ACC_CPU_B_TBB_T_SEQ_ENABLED)
```

Comment:
Why are the Omp5 and OpenAcc defines not in this list?

Reply:
Because they each require their own `Dev...` type, so they are arranged in the same style as CUDA and HIP.